engines/sci/graphics/frameout.cpp

```diff
@@ -1334,14 +1366,15 @@ void GfxFrameout::kernelSetNowSeen(const reg_t screenItemObject) const {
 
 	ScreenItem *screenItem = plane->_screenItemList.findByObject(screenItemObject);
 	if (screenItem == nullptr) {
-		error("kSetNowSeen: Screen item %04x:%04x not found in plane %04x:%04x", PRINT_REG(screenItemObject), PRINT_REG(planeObject));
+		return false;
```

Comment:
We should at the very least output a warning, so that errors are recognizable as such.
For SCI2/early games maybe we should even keep the error. It's not about running games properly, but being able to detect errors and not just skip over them, which could then cause all sorts of weird issues and could take ages to actually track down.

With the new code kSetNowSeen calls with invalid screen items could cause all sorts of issues, which would then be invisible to us and you don't know if the current code is perfect - and that even including the games, where such an issue in the original interpreter would literally "crash" the game (and where it would definitely be either a script error or our own fault). I don't see any issues with adding a warning. When someone plays through the games, that use the new code, we would at the very least see if such situations actually occur, could then write them down and maybe make sure that it's not our fault but how the games actually work.

And well, we could also just ignore the warnings anyway. But at least they would be there. If such a warning would happen all the time in those other games, then that's a different matter. But that wouldn't change SCI2/early behavior.